src/components/keymap/header/keymap-header.component.ts

```diff
@@ -14,10 +14,9 @@ import { KeymapActions } from '../../../store/actions';
 })
 export class KeymapHeaderComponent {
     @Input() keymap: Keymap;
+    @Output() deleted: EventEmitter<boolean> = new EventEmitter();
```

Comment:
As I am more deeply in how it works I think it should be named to `deleteClick` with type `EventEmitter<void>` and move the `AppState `dispatching logic to the `Keymap(Edit)Component`.